ako-operator/helm/ako-operator/crds/ako.vmware.com_akoconfigs.yaml

```diff
@@ -150,6 +163,10 @@ spec:
                     description: DefaultIngController specifies whether AKO controller
                       is the default ingress controller
                     type: boolean
+                  enableMCI:
```

Comment:
@DixitAakash: Do not include it.

Reply:
Removed from AKOConfig CRD Spec but added to ako configmap.